mongodbatlas/data_source_mongodbatlas_cluster.go

```diff
@@ -337,8 +337,8 @@ func dataSourceMongoDBAtlasClusterRead(ctx context.Context, d *schema.ResourceDa
 			Target:     []string{"PRIVATE_ENDPOINTS_EXISTS", "NORMAL"},
 			Refresh:    datasourceClusterPrivateEndpointRefreshFunc(clusterName, projectID, conn),
 			Timeout:    10 * time.Minute,
-			MinTimeout: 1 * time.Minute,
-			Delay:      3 * time.Minute,
+			MinTimeout: 5 * time.Second,
+			Delay:      3 * time.Second,
```

Comment:
Is there a way to skip they delay unless absolutely needed like in the case of privatelink?  I mean if it would return in a second it doesn't seem to make sense to delay it arbitrarily.

Reply:
Yes, I think it's possible to skip the delay, let me try and push the changes if it works